charmcraft/commands/pack.py

```diff
@@ -100,6 +100,11 @@ def fill_parser(self, parser):
             type=pathlib.Path,
             help="Dump measurements to the specified file",
         )
+        parser.add_argument(
+            "--no-use-shared-cache",
```

Comment:
`--disable-shared-cache` can be an alternative if we're not using the `--no` pattern elsewhere.

Reply:
we agreed to remove this for now

Reply:
in the future, in should be `enable/disable-shared-cache`

Reply:
I was thinking use `BooleanOptionalAction`, but that one only added in Python 3.9.
So now I just change it to `--disable-shared-cache`